### ISSUE TITLE
[Screen Reader-Social Embedded-Report Post]: TalkBack announces incorrect state for the radio buttons.

### ISSUE THREAD

**Issue by billycamp** [Microsoft/EmbeddedSocial-Android-SDK#18]
"Priority: High

Repro Steps
1. Launch the One Bus Away App.
2. Navigate to ""Sign In"" link under Hamburger link and press ENTER.
3. Sign in screen showing all currently supported identity service providers will appear.
4. Use any service provider for log in.
5. Select any location bubble and see the given arrivals.
6. Navigate to ""Join Discussion"" button and press ENTER.
7. Navigate to ellipses next to the post (topic) and press ENTER.
8. Press ENTER key on ""Report Post"" link and press ENTER to open ""Report an Issue"" page.
9. Swipe to navigae to any selected radio button.
10. Verify the speech of talkback for the selected radio button.

Expected
Talkback should announce the state as ""Selected"" for selected radio buttons, so that screen reader users can have a clear idea, that radio buttons are given for the single selection on the page.

Actual
Talkback announces ""Checked"" state for the selected radio button. It might confuse screen reader users whether they can make multiple selection in case of checkboxes with ""checked"" state or they can make only one selection in case of radio button with ""Selected"" state.

MAS Rule: MAS4.2.1

Impact
If talkback announces the incorrect state as ""Checked""for radio buttons, then screen reader users will get confused to identify the radio buttons for the selection on the page.

Test Environment
OS: Android
Device Model No. : Nexus 5
Android Version : 6.0.1
App: Social Embedded
App Version : v2.2.3 (84)
Tool : Talk Back"
